lib/BiochemistryAPI/utils.py

```diff
@@ -93,44 +95,61 @@ def _get_mol(structure):
     return mol
```

Comment:
This is a compound that I got either from the URL or from `data/compounds.tsv` ... forget
```
OrderedDict([('id', 'cpd00003'),
             ('abbreviation', 'nad'),
             ('name', 'NAD'),
             ('formula', 'C21H26N7O14P2'),
             ('mass', '662.0'),
             ('source', 'Primary Database'),
             ('structure', 'BAWFJGJZGIEFAR-NNYOXOHSSA-M'),
             ('charge', '-1'),
             ('is_core', '1'),
             ('is_obsolete', '0'),
             ('is_cofactor', '0'),
             ('deltaG', '-286.41'),
             ('deltaGErr', '1.59'),
             ('pka', '1:6:11.94;1:17:1.85;1:18:2.28;1:25:11.38;1:35:12.70'),
             ('pkb', '1:6:-4.22;1:35:-3.85;1:37:-1.05;1:41:4.93;1:43:-7.46'),
             ('aliases',
              "['Name: DPN; DPN+; DPN-ox; Diphosphopyridine nucleotide; NAD; NAD+; NAD-ox; NAD-oxidized; Nadide; Nicotinamide adenine dinucleotide; Nicotinamideadeninedinucleotide; beta-NAD+; beta-nicotinamide adenine dinucleotide; coenzyme I; diphosphopyridine nucleotide; diphosphopyridine nucleotide oxidized; nicotinamide adenine dinucleotide; nicotinamide adenine dinucleotide oxidized', 'AraCyc: NAD', 'BiGG: nad', 'BrachyCyc: NAD', 'KEGG: C00003', 'MetaCyc: NAD']")])
```
As you can see it uses InChIKey

Reply:
Oooh, that's a good catch. We updated the original biochemistry to only use InChIKey and SMILES because InChI was too long, but you can't do the structural matching and manipulation in here with InChIKey, so I'll have to change and test this:
https://github.com/kbaseapps/BiochemistryAPI/blob/master/data/Fetch_Updated_Data_from_ModelSEED.py#L19

Reply:
I've made a note of this and will address it in the near future as I want to get the testing working first